src/loops/simple_programs_with_repitition/easy/WordCount.java

```diff
@@ -0,0 +1,24 @@
+package loops.simple_programs_with_repitition.easy;
+
+/*
+Write a program that will ask the user for a sentence and 
```

Comment:
Again, there is no sample code for the question, so the clause will mark them wrong, and we have not provided a correct formating for them. Code looks good though

Reply:
Nevermind, you should not need sample code for this one